ui/src/views/compute/wizard/ComputeSelection.vue

```diff
@@ -25,7 +25,7 @@
             :validate-status="errors.cpu.status"
             :help="errors.cpu.message">
             <a-row :gutter="12">
-              <a-col :md="10" :lg="10" v-show="isConstrained">
+              <a-col :md="10" :lg="10" v-show="isConstrained && maxCpu && isNaN(maxCpu)">
```

Comment:
@utchoang is this show condition correct here. For memory you have used reversed for isNaN which probably makes sense

Reply:
@shwstppr Yes. I'm wrong. Thanks